charts/prometheus-blackbox-exporter/values.yaml

```diff
@@ -88,6 +88,9 @@ runAsGroup: 1000
 readOnlyRootFilesystem: true
 runAsNonRoot: true
 allowPrivilegeEscalation: false
+capabilities:
+  dropAllCapabilities: true
+  allowIcmp: false
```

Comment:
this introduces a breaking change.

imho it would be better to implement it like this, keeping the current default values: 
https://github.com/prometheus-community/helm-charts/blob/0c70d2b4855fc096716afc07b648f6230207409d/charts/prometheus-json-exporter/values.yaml#L28-L40

Reply:
You're right, we can simplify it, but it takes `allowIcmp` for other resources so that is cause I prefer to continue with this approach. So if you agree, we can remove the capabilities indentation and leave all as root level to avoid breaking changes